README.md

```diff
@@ -144,6 +153,12 @@ You can configure and manage KoP based on your requirements. Check the following
 - [Manage KoP with the [Envoy](https://www.envoyproxy.io) proxy](docs/envoy-proxy.md)
 - [Implementation: How to converse Pulsar and Kafka](docs/implementation.md)
 
+The following are important information when you configure and use KoP.
+
+- Set both [retention and (time to live) TTL](http://pulsar.apache.org/docs/en/cookbooks-retention-expiry/) as `7 days` for KoP topics. If you only configure retention without configuring TTL, all messages of KoP topics cannot be deleted because KoP does not update a durable cursor.
```

Comment:
I think it's not proper to give a specific config value like `7 days` here, I said `7 days` just because Kafka's default retention time is 7 days, see https://kafka.apache.org/documentation/#topicconfigs_retention.ms.